internal/verifications/naming/get_interfaces.go

```diff
@@ -75,6 +68,103 @@ func retrieveMatchingInterfaces(
 	return interfaces
 }
 
+func retrieveMethods(currentInterface *InterfaceDescription, it *ast.InterfaceType, data []byte, node *ast.File) {
+	for _, field := range it.Methods.List {
+		switch tp := field.Type.(type) {
+		case *ast.FuncType:
+			retrieveNormalMethods(currentInterface, field, tp, data)
+		case *ast.Ident:
+			retrieveMethodsFromEmbeddedInterface(currentInterface, tp, data, node)
+		case *ast.IndexExpr:
+			retrieveGenericsMethods(currentInterface, tp, data, node)
+		case *ast.SelectorExpr:
+			retrieveOtherPackageMethods(currentInterface, tp, node)
+		}
```

Comment:
could you please add a default case here, which panics with information about the used fiedl type?

Reply:
Just added. Thanks for the suggestion!